### PR TITLE
Add question about immutability

### PR DESCRIPTION
**Pull request by zeitiger**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -138,6 +138,9 @@ duplicate([1,2,3,4,5]); // [1,2,3,4,5,1,2,3,4,5]
 * What is the extent of your experience with Promises and/or their polyfills?
 * What are the pros and cons of using Promises instead of callbacks?
 * What are some of the advantages/disadvantages of writing JavaScript code in a language that compiles to JavaScript?
+* Explain mutability and immutability on example of JS basic types (string, object, array, number, etc.)
```
Comment (darcyclarke) [inline README.md]: Maybe fix this up to be more like:
> Explain mutable and immutable objects.
  * What is an example of an immutable object in JavaScript?
  * What are the pros and cons of immutability?
  * How can you achieve immutability in your own code?

Referencing the basic types seems to give away an answer you could ask for immutable objects.

Comment (zeitiger) [inline README.md]: :+1:

**Comment by darcyclarke** [comment]
Looks good @zeitiger! Can you update your branch so we can merge? There's been changes to master since you made your initial pull request. A pull/merge should fix that up.

**Comment by appleboy** [comment]
@zeitiger rebase master on your current branch.

```
$ git rebase maste
```

**Comment by zeitiger** [comment]
rebased

**Comment by richgilbank** [comment]
Looks like there were some merge conflicts, so it's showing multiple commits currently. Mind taking another crack at it?

**Comment by arthurvr** [comment]
> Looks like there were some merge conflicts, so it's showing multiple commits currently. Mind taking another crack at it?

ping?

**Comment by zeitiger** [comment]
I close this and transfer the change to a new pull request

**Comment by arthurvr** [comment]
@zeitiger Thanks for picking this up! Though you can just make the edit to this PR, in fact.

**Pull request closed without merging**